manifests/reverse_proxy.pp

```diff
@@ -54,12 +54,12 @@
     ssl_options            => ['+StdEnvVars', '+ExportCertData', '+FakeBasicAuth'],
     ssl                    => true,
     ssl_proxyengine        => true,
-    ssl_proxy_ca_cert      => $certs::ca_cert,
+    ssl_proxy_ca_cert      => $certs::apache::apache_client_ca_cert,
```

Comment:
Now I'm debating if this should somehow be referring to `$certs::foreman_proxy` because Foreman Proxy is also configured with a CA to trust. This would closely relate to `ssl_proxy_machine_cert`. Have you had any thoughts in that direction?